### PR TITLE
LPS-194889 Do not scroll when hovering on an item

### PR DESCRIPTION
**Pull request by veroglez**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by veroglez** [comment]
ci:test:sf

File: modules/apps/layout/layout-reports-web/src/main/resources/META-INF/resources/js/components/render_times/FragmentList.tsx [hunk 1]
```diff
@@ -75,11 +77,13 @@ export default function FragmentList({
 			style: {left: rect.x, top: rect.y + rect.height + window.scrollY},
 		});
 
-		fragment?.scrollIntoView?.({
-			behavior: 'smooth',
-			block: 'center',
-			inline: 'nearest',
-		});
+		if (!disableScroll) {
```
Comment (victorg1991) [inline modules/apps/layout/layout-reports-web/src/main/resources/META-INF/resources/js/components/render_times/FragmentList.tsx:80]: Can we change it to a positive variable like `scrollToItem` or `enableScroll`.  Truthy conditions tends to be read better :)

**Comment by victorg1991** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:test:stable

**Comment by Tim-Cao** [comment]
ci:forward:force

**Pull request closed without merging**